### PR TITLE
Remove Pod Network from a VM

### PR DESCRIPTION
**Pull request by moadqassem**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note

```

**Documentation**:

```documentation

```

### CODE REVIEW

**Comment by cnvergence** [comment]
I see this config was added after an issue:
https://github.com/kubermatic/machine-controller/issues/1202

File: pkg/cloudprovider/provider/kubevirt/provider.go [hunk 1]
```diff
@@ -788,6 +779,7 @@ func randomMacAddressGetter() (string, error) {
 	return mac.String(), nil
 }
 
+// TODO(mq): refactor networking in VMs
 func defaultBridgeNetwork(macAddressGetter macAddressGetter) (*kubevirtv1.Interface, error) {
```
Comment (cnvergence) [inline pkg/cloudprovider/provider/kubevirt/provider.go:783]: needs to be removed, because of the lint

**Comment by moadqassem** [comment]
This PR is not needed for now as this needs to be investigated further,

**Comment by moadqassem** [comment]
This PR is not needed for now as this needs to be investigated further.

**Pull request closed without merging**